src/main/java/com/minecolonies/coremod/entity/ai/citizen/miner/EntityAIStructureMiner.java

```diff
@@ -797,10 +821,24 @@ public void executeSpecificCompleteActions()
             minerBuilding.setCurrentLevel(minerBuilding.getNumberOfLevels());
             minerBuilding.resetStartingLevelShaft();
         }
+        super.executeSpecificCompleteActions();
+
         //Send out update to client
         getOwnBuilding().markDirty();
-
-        job.setStructure(null);
+         job.setStructure(null);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 9. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)